schema.graphql

```diff
@@ -24,7 +24,7 @@ type Item @entity {
   owner: String
 }
 
-type BalanceTransfer @entity {
+type BalanceTransfer @entity @compositeIndexes(fields: [["from", "to", "amount"], ["from", "to", "timestamp"], ["from", "to"]]) {
```

Comment:
This composite indexing doesn't seem to be able to accelerate queries such as from not equal to allocations to dao or company reserve. Correct me if I'm wrong. I'm not against keeping it if it's making our other usual queries faster. However to satisfy https://linear.app/nodle/issue/CHA-137 you may need to create a new entity such as 
```
type TransferToTreasury @entity {
id : ID!
from: String! @index
to: String! @index # This field is still needed because we have a number of treasury accounts and not just one
timestamp: Float
blockNumber: Int 
}
```
The other option is you add another field to `BalanceTransfer` such as `nonRewardToTreasury: Boolean @index` to make it clear which entities are "transfers to one of our treasuries but not coming from the allocations".

Reply:
@aliXsed could you give me all the treasury accounts, or it is a constant in the chain?

Reply:
Company Reserve: 4jbtsgNhpGB2vH7xTjpZVzZLy7W4sFyxjvD45x7X1m6BSiGx
International Reserve: 4jbtsgNhpGB2voF5dZzKQ2tphWLjV48HSkfQwmWqNn3qa4rv
USA Reserve: 4jbtsgNhpGB2voKv8rRSJYTAohbnHE5oVZ4DejZBEQVT5o86
DAO Reserve: 4jbtsgNhpGB2NtbdLN3xkB2urHo5JboKkSAjcBM4s3SzQdUt
Allocations: 4jbtsgNhpGAzdEGrKRb7g8Mq4ToNUpBVxeye942tWfG3gcYi